src/redux/reducers/cats.js

```diff
@@ -18,7 +21,13 @@ export default function cats(state = initialState, action) {
   switch (action.type) {
     case 'cats/RANDOM_CAT':
       return { ...state, cat: randomCat() }
-    default:
       return state
+
+    case 'cats/ADOPT_CAT':
+      return { 
+          ...state,
+          catAdopt: state.adopt.concat(action.ADOPT_CAT) }
```

Comment:
The key should be `adoptedCats` instead of `catAdopt`, as we are trying to write over our empty array in state.

`state.adopt.concat(action.ADOPT_CAT)` is close. I would first try and send the cat you are adopting with the `adoptCat` action as the payload, then you can look at adding it to the array.